room-states.typ

```diff
@@ -1,3 +1,5 @@
+#import "@preview/in-dexter:0.0.5": *
```

Comment:
I thought the import was only needed in main?

Reply:
I was wrong. We'll import it on an as-needed basis.